src/valkey-cli.c

```diff
@@ -2338,21 +2338,22 @@ static int cliSendCommand(int argc, char **argv, long repeat) {
     if (context == NULL) return REDIS_ERR;
 
     output_raw = 0;
-    if (!strcasecmp(command, "info") || !strcasecmp(command, "lolwut") ||
-        (argc >= 2 && !strcasecmp(command, "debug") && !strcasecmp(argv[1], "htstats")) ||
-        (argc >= 2 && !strcasecmp(command, "debug") && !strcasecmp(argv[1], "htstats-key")) ||
-        (argc >= 2 && !strcasecmp(command, "debug") && !strcasecmp(argv[1], "client-eviction")) ||
-        (argc >= 2 && !strcasecmp(command, "memory") &&
-         (!strcasecmp(argv[1], "malloc-stats") || !strcasecmp(argv[1], "doctor"))) ||
-        (argc == 2 && !strcasecmp(command, "cluster") &&
-         (!strcasecmp(argv[1], "nodes") || !strcasecmp(argv[1], "info"))) ||
-        (argc >= 2 && !strcasecmp(command, "client") &&
-         (!strcasecmp(argv[1], "list") || !strcasecmp(argv[1], "info"))) ||
-        (argc == 3 && !strcasecmp(command, "latency") && !strcasecmp(argv[1], "graph")) ||
-        (argc == 2 && !strcasecmp(command, "latency") && !strcasecmp(argv[1], "doctor")) ||
-        /* Format PROXY INFO command for Cluster Proxy:
-         * https://github.com/artix75/redis-cluster-proxy */
-        (argc >= 2 && !strcasecmp(command, "proxy") && !strcasecmp(argv[1], "info"))) {
+    if (!config.in_multi &&
+        (!strcasecmp(command, "info") || !strcasecmp(command, "lolwut") ||
+         (argc >= 2 && !strcasecmp(command, "debug") && !strcasecmp(argv[1], "htstats")) ||
+         (argc >= 2 && !strcasecmp(command, "debug") && !strcasecmp(argv[1], "htstats-key")) ||
+         (argc >= 2 && !strcasecmp(command, "debug") && !strcasecmp(argv[1], "client-eviction")) ||
+         (argc >= 2 && !strcasecmp(command, "memory") &&
+          (!strcasecmp(argv[1], "malloc-stats") || !strcasecmp(argv[1], "doctor"))) ||
+         (argc == 2 && !strcasecmp(command, "cluster") &&
+          (!strcasecmp(argv[1], "nodes") || !strcasecmp(argv[1], "info"))) ||
+         (argc >= 2 && !strcasecmp(command, "client") &&
+          (!strcasecmp(argv[1], "list") || !strcasecmp(argv[1], "info"))) ||
+         (argc == 3 && !strcasecmp(command, "latency") && !strcasecmp(argv[1], "graph")) ||
+         (argc == 2 && !strcasecmp(command, "latency") && !strcasecmp(argv[1], "doctor")) ||
+         /* Format PROXY INFO command for Cluster Proxy:
+          * https://github.com/artix75/redis-cluster-proxy */
+         (argc >= 2 && !strcasecmp(command, "proxy") && !strcasecmp(argv[1], "info")))) {
         output_raw = 1;
     }
 
```

Comment:
can we correct the `output_raw` in here? something like this:
```
if (config.in_multi) output_raw = 0; /* multi must in xxx mode. */
```

Reply:
I assume readability, as it has the same result. I will say I would prefer we add a comment somewhere to include why multi does this. My hot take was going to be let's just do:

```
if (!config.in_multi) output_raw = 1; /* In a multi block, commands will return status instead of verbatim. */
```

I don't feel all that strongly though.

Reply:
You meant:
```
if (config.in_multi) output_raw = 0; /* In a multi block, commands will return status instead of verbatim. */
```
?

I want to skip big if. 
See https://github.com/valkey-io/valkey/pull/1782/commits/61559bfcb69b56edcddfb8ecd61e5025902f4d19


This function could be more readable; I can refactor it in a separate PR.

Reply:
I meant my suggestion inside the block of text, so it minimizes the number of lines changed. I'm OK with this though.